docs/_includes/supporters.md

```diff
@@ -2,13 +2,36 @@
 
 Use Mocha at Work? Ask your manager or marketing team if they'd help [support](https://opencollective.com/mochajs#support) our project. Your company's logo will also be displayed on [npmjs.com](http://npmjs.com/package/mocha) and our [GitHub repository](https://github.com/mochajs/mocha#sponsors).
 
+<style>
+  .sponsor {
+    -sprite-selector-for-group: sponsors;
+    -sprite-location: url(/images/sprite-sponsors.png?pngquant);
+    -sprite-image-format: png;
+    height: 64px;
+  }
+  .backer {
+    -sprite-selector-for-group: backers;
+    -sprite-location: url(/images/sprite-backers.png?pngquant);
+    -sprite-image-format: png;
+    width: 32px;
+    height: 32px;
+  }
+
+</style>
```

Comment:
does this want to be extracted into `style.css`?

Reply:
Yeah. I think I'll do that. That should also remove an artifact where the page will jump on reload when you're scolled down a bit. Currently there's a render flush when the browser hits this style tag because it becomes external. That makes the page scrollheight briefly be equal to the offset of this tag, which makes the page jump up to the top, then immediately down again once the async css for the sprites has loaded